src/Blog/BlogCardContent.js

```diff
@@ -1,71 +1,82 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { format } from 'date-fns';
+import React from "react";
+import PropTypes from "prop-types";
+import { format } from "date-fns";
 
 const propTypes = {
-    contentItem: PropTypes.object.isRequired
+  contentItem: PropTypes.object.isRequired
 };
 
 const getCategoryImage = category => {
-    switch (category) {
-        case 'News':
-            return '/sebin/v/f/blog-default-news.jpg';
-        case 'Programming and Events':
-            return '/sebin/t/i/blog-default-events.jpg';
-        case 'Collection and Materials':
-        default:
-            return '/sebin/d/w/blog-default-reviews.jpg';
-    }
+  switch (category) {
+    case "News":
+      return "/sebin/v/f/blog-default-news.jpg";
+    case "Programming and Events":
+      return "/sebin/t/i/blog-default-events.jpg";
+    case "Collection and Materials":
+    default:
+      return "/sebin/d/w/blog-default-reviews.jpg";
+  }
 };
 
 export default class BlogCardContent extends React.Component {
-    render() {
-        const {
-            Title,
-            PublishedDate,
-            Author,
-            ShortDescription,
-            Category,
-            CategoryIcon,
-            Link,
-            ImageUrl,
-            ImageAlt
-        } = this.props.contentItem;
+  render() {
+    const {
+      Title,
+      PublishedDate,
+      Author,
+      ShortDescription,
+      Category,
+      CategoryIcon,
+      Link,
+      ImageUrl,
+      ImageAlt,
+      Pin_Blog_Entry
```

Comment:
this is going to change from `Pin_Blog_Entry` to `IsFeatured`

Reply:
modified to IsFeatured